docs/src/examples/components/Indicator/index.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+import Types from './Types'
+
+const IndicatorExamples = () => (
+  <div>
```

Comment:
```suggestion
  <>
```
We can use there Fragment API, `div` is a legacy from SUIR